examples/chip-tool/config/PersistentStorage.cpp

```diff
@@ -77,6 +77,28 @@ std::string Base64ToString(const std::string & b64Value)
     return std::string(reinterpret_cast<const char *>(buffer.get()), len);
 }
 
+void AdjustConfig(Sections & sections)
+{
+    Section section = sections[kDefaultSectionName];
+    Section newSection;
+
+    for (auto it = section.begin(); it != section.end(); it++)
+    {
+        std::string key    = it->first;
+        const char * value = it->second.c_str();
+
+        while (value[0] == '=')
```

Comment:
So this is assuming that values don't actually ever start with '=', right?  I guess that's true because we base64-encode the values?  Might be worth documenting that...